mqtt/mqtt-broker/src/lib.rs

```diff
@@ -81,7 +81,7 @@ impl Display for ClientId {
     }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct ClientInfo {
     peer_addr: SocketAddr,
     auth_id: AuthId,
```

Comment:
since session will always have last used client info, lets add client_id into this struct? it should simplify code a little bit